test/tools/runner/config.ts

```diff
@@ -356,7 +358,7 @@ export class TestConfiguration {
   }
 
   // Accessors and methods Client-Side Encryption
-  get mongodbClientEncryption() {
+  get mongodbClientEncryption(): typeof import('mongodb-client-encryption') {
```

Comment:
Out of curiosity, why was this needed?

Reply:
Not needed, I sync-ed this file with v5 which has this change. It just makes autocomplete work when you're writing an FLE test, which is nice, but unrelated to this work